commit c47346f5f7a195193159d0988dc843911982cc33
Author: q66 <q66@chimera-linux.org>
Date:   Sat Mar 23 20:03:16 2024 +0100

    disable gdbus-connection-slow tests
    
    This times out on aarch64.

diff --git a/gio/tests/meson.build b/gio/tests/meson.build
index 232ecca..4a799a1 100644
--- a/gio/tests/meson.build
+++ b/gio/tests/meson.build
@@ -439,12 +439,6 @@ if host_machine.system() != 'windows'
         'extra_sources' : extra_sources,
         'extra_programs': extra_programs,
       },
-      'gdbus-connection-slow' : {
-        'extra_sources' : extra_sources,
-        'extra_programs': extra_programs + ['gdbus-connection-flush-helper'],
-        # FIXME: https://gitlab.gnome.org/GNOME/glib/-/issues/3148
-        'can_fail' : host_system == 'gnu',
-      },
       'gdbus-error' : {'extra_sources' : extra_sources},
       'gdbus-exit-on-close' : {'extra_sources' : extra_sources},
       'gdbus-export' : {

